SoftLayer/CLI/image/datacenter.py

```diff
@@ -0,0 +1,29 @@
+"""Edit details of an image."""
+# :license: MIT, see LICENSE for more details.
+
+import click
+
+import SoftLayer
+from SoftLayer.CLI import environment
+from SoftLayer.CLI import helpers
+
+
+@click.command()
+@click.argument('identifier')
+@click.option('--add/--remove',
+              default=False,
```

Comment:
default to True, so the default behavior will be to add dcs, and put this on the same line as `--add/--remove`, help can be on its own line though.